akka-actor/src/main/java/akka/dispatch/affinity/OnSpinWait.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.dispatch.affinity;
+
+import akka.util.Unsafe;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import scala.util.control.NonFatal;
+import static java.lang.invoke.MethodType.methodType;
+
+final class OnSpinWait {
```

Comment:
Let's keep it internal, at least documented as such:
```
/**
 * INTERNAL API
 */
@InternalApi
```

Reply:
Agreed